tests/convert/test_area.py

```diff
@@ -189,81 +190,81 @@ def test_convert_equal(self, regions_rect):
         regions_rect_alt.name = 'rect_alt'
         rreg.register(regions_rect_alt)
 
-        data = {'zero': 1}
+        data = np.ones(1)
         converted = rreg.convert(data, 'rect', 'rect_alt')
-        assert data == converted
+        assert all(data == converted)
 
     def test_convert_to_half(self, regions_rect, regions_half_squares):
         rreg = RegionRegister()
         rreg.register(regions_rect)
         rreg.register(regions_half_squares)
 
-        data = {'zero': 1}
+        data = np.ones(1)
         converted = rreg.convert(data, 'rect', 'half_squares')
-        expected = {'a': 0.5, 'b': 0.5}
-        assert converted == expected
+        expected = np.ones(2) / 2
+        assert all(converted == expected)
```

Comment:
`numpy.testing` contains helper methods `assert_equal` and all_close` which handle array comparison